tests/Aspire.StackExchange.Redis.Tests/ConformanceTests.cs

```diff
@@ -1,26 +1,30 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Aspire.Components.ConformanceTests;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using StackExchange.Redis;
+using Xunit;
 
 namespace Aspire.StackExchange.Redis.Tests;
 
-public class ConformanceTests : ConformanceTests<IConnectionMultiplexer, StackExchangeRedisSettings>
+public class ConformanceTests : ConformanceTests<IConnectionMultiplexer, StackExchangeRedisSettings>, IClassFixture<RedisContainerFixture>
 {
+    private readonly RedisContainerFixture _containerFixture;
+
     protected override ServiceLifetime ServiceLifetime => ServiceLifetime.Singleton;
 
     // IConnectionMultiplexer can be created only via call to ConnectionMultiplexer.Connect
     protected override bool CanCreateClientWithoutConnectingToServer => false;
 
-    protected override bool CanConnectToServer => AspireRedisHelpers.CanConnectToServer;
+    protected override bool CanConnectToServer => RequiresDockerTheoryAttribute.IsSupported;
 
     protected override bool SupportsKeyedRegistrations => true;
 
-    protected override string[] RequiredLogCategories => ["StackExchange.Redis"];
+    protected override string[] RequiredLogCategories => ["StackExchange.Redis.ConnectionMultiplexer"];
```

Comment:
This was changed to match what we actually get when the test is run.

Reply:
Interesting, how come were these not failing in the past? Or were they failing and we just weren't running them?

Reply:
These weren't run on CI before. I don't know how they passed if they were run locally.